pulsar/consumer_impl.go

```diff
@@ -319,6 +302,15 @@ func (c *consumer) Close() {
 	})
 }
 
+func (c *consumer) Seek(msgID MessageID) error {
```

Comment:
If the consumer is partitioned we should return an error

Reply:
How can I know this consumer is partitioned?

Reply:
If there is more than 1 consumer in the list then it's a partitioned topic. If topic is partitioned and has only 1 partition or the topic is not partitioned then there will be 1 consumer which is kind of the same thing.

Reply:
Done.